src/styles/index.ts

```diff
@@ -1388,14 +1388,14 @@ const styles = (theme: ThemeColors) =>
         formHelp: {
             color: theme.textSupporting,
             fontSize: variables.fontSizeLabel,
-            lineHeight: variables.lineHeightLarge,
+            lineHeight: variables.lineHeightNormal,
             marginBottom: 4,
         },
 
         formError: {
             color: theme.textError,
             fontSize: variables.fontSizeLabel,
-            lineHeight: variables.formErrorLineHeight,
+            lineHeight: variables.lineHeightNormal,
```

Comment:
@shawnborton formErrorLineHeight - getValueUsingPixelRatio(18, 23),
lineHeightNormal: getValueUsingPixelRatio(16, 21),

what does getValuesUsingPixelRatio does:

> Calculate the fontSize, lineHeight and padding when the device font size is changed, In most cases users do not change their device font size so PixelRatio.getFontScale() = 1 and this method always returns the defaultValue (first param). When the device font size increases/decreases, the PixelRatio.getFontScale() value increases/decreases as well. This means that if you have text and its 'fontSize' is 19, the device font size changed to the 5th level on the iOS slider and the actual fontSize is 19 * PixelRatio.getFontScale() = 19 * 1.11 = 21.09. Since we are disallowing font scaling we need to calculate it manually. We calculate it with: PixelRatio.getFontScale() * defaultValue > maxValue ? maxValue : defaultValue * PixelRatio.getFontScale() This means that the fontSize is increased/decreased when the device font size changes up to maxValue (second param)

Reply:
Got it. I mostly just think the naming convention feels strange, but that's a separate conversation I suppose.

For example, our "normal" font size is 15px, and it should have a line height of 20px. But it's strange that we call the 20px line height `lineHeightLarge`

Our "label" font size is 13px, and it should have a line height of 16px. But it's strange that we call the 16px line height `lineHeightNormal`

Reply:
yeah lol - some namings looks strange:
<img width="706" alt="Screenshot 2024-05-06 at 18 04 47" src="https://github.com/Expensify/App/assets/28352309/e8afb20a-afeb-4964-a259-28674a925aff">